### ISSUE TITLE
content images for training the multi-style model

### ISSUE THREAD

**Issue by salopge** [Yijunmaverick/MultiTextureSynthesis#1]
Hello @Yijunmaverick 

Could you share or explain content images used for your multi style transfer model?
I could not find any clues from the README description and your paper.

Thanks,
@salopge

**Comment by Yijunmaverick** [comment]
Hi @salopge, thanks for your interests. You can collect several thousand of content images as the content is relatively easier to reconstruct (2 pooling layers only). Any image can be a content image. For me, I use 2k images randomly selected from CelebA and AVA dataset.

**Comment by salopge** [comment]
@Yijunmaverick, thanks for your reply.

BTW, can you share the training parameters  for your pre-trained style transfer model with 1k styles? (hyper parameters, the number of iterations, etc)
I had trained a my own model with your code but I was not able to achieve the same quality as yours.

Thanks.

**Comment by suke27** [comment]
Hello @Yijunmaverick
I collect 6000 contents file and 30  style images, found during training the diversity loss always 0
I use default parameter, could you help to check.
Thank you!

style =         025
Iter =  3767    Style =         3781737.460041  Content =       159997.18125    Diversity =     0
content =       11735
style =         026
flip content
Iter =  3768    Style =         2978913.1427765 Content =       137886.328125   Diversity =     0
content =       13818
style =         030
Iter =  3769    Style =         3751073.3333588 Content =       176960.08125    Diversity =     0
content =       14715
style =         010
flip content
Iter =  3770    Style =         1270537.5136614 Content =       133004.559375   Diversity =     0
content =       12110
style =         025
Iter =  3771    Style =         3835246.413517  Content =       151061.4375     Diversity =     0
content =       14520
style =         026
flip content
Iter =  3772    Style =         3312022.2510338 Content =       144280.5375     Diversity =     0
content =       11696
style =         030
Iter =  3773    Style =         3103662.6737595 Content =       196270.06875    Diversity =     0
content =       15355
style =         010
Iter =  3774    Style =         1140389.3590987 Content =       124324.921875   Diversity =     0
content =       10339
style =         025
Iter =  3775    Style =         4470940.9083366 Content =       142396.6875     Diversity =     0
content =       10367
style =         026
Iter =  3776    Style =         3622527.909565  Content =       132104.1375     Diversity =     0
content =       16178
style =         030
Iter =  3777    Style =         2930067.8323746 Content =       182809.6875     Diversity =     0
content =       14833
style =         010
Iter =  3778    Style =         1503847.9936659 Content =       125900.634375   Diversity =     0
content =       14474
style =         025
Iter =  3779    Style =         3601357.828331  Content =       135513.384375   Diversity =     0
content =       13488
style =         026
Iter =  3780    Style =         3781062.3197079 Content =       134112.421875   Diversity =     0
content =       12172
style =         030
flip content
Iter =  3781    Style =         2625414.586401  Content =       226077.5625     Diversity =     0
content =       10698
style =         010
flip content
Iter =  3782    Style =         1181360.8678401 Content =       135482.11875    Diversity =     0
content =       15443
style =         025
flip content
Iter =  3783    Style =         3793514.4950867 Content =       139716.13125    Diversity =     0
content =       15122
style =         026
Iter =  3784    Style =         3788427.0775795 Content =       130540.359375   Diversity =     0
content =       14351
style =         030
flip content
Iter =  3785    Style =         3021783.6183071 Content =       207036.4125     Diversity =     0
content =       14733
style =         010
Iter =  3786    Style =         1132884.8307073 Content =       108309.375      Diversity =     0
content =       10999
style =         025

**Comment by Yijunmaverick** [comment]
@suke27 Because I set `diversity_weight=0` by default in `multi_style_transfer_train.lua` :-)

**Comment by suke27** [comment]
oh, yes, 
diversity loss is for same style loss computing with different noise input. is it influence to result during 1000 styles training?

**Comment by Yijunmaverick** [comment]
@suke27 Generally it will make the training slower, especially when learning a large number of styles. That's why I set it as 0. I suggest you use a smaller diversity weight (e.g., -0.1). The results should be similar.

**Comment by suke27** [comment]
ok, thank you for your reply

**Comment by suke27** [comment]
Hi, ^^, during training, I found cmd:option('-num_iterations', 1000000000), is it your style 1000 parameter? that is a great number, how long your training cost. I just select 30 styles with 2k contents images

**Comment by Yijunmaverick** [comment]
@suke27 It's just a random number, you can monitor the loss/results and terminate anytime.

**Comment by suke27** [comment]
@Yijunmaverick , I have check transfer result for MultiTextureSynthesis , just based on result, seems MultiTextureSynthesis save texture and brush info, but for color space, not transfer well. loot at below result, the transfer result only capture the yellow color, since there is only texture loss to adjust. is there some way to adjust color space?
![4](https://user-images.githubusercontent.com/30656046/36888543-d20423ca-1ded-11e8-8fcf-31a10db07084.jpg)
